src/index.ts

```diff
@@ -8,35 +8,38 @@
  * Learn more at https://developers.cloudflare.com/workers/
  */
 
+import { Env, Feedback } from "./types";
+import { APIEmbed, RESTPostAPIWebhookWithTokenJSONBody } from 'discord-api-types/v10';
+
 import { OpenAIApi, Configuration } from "openai";
 import fetchAdapter from "@vespaiach/axios-fetch-adapter";
 
-async function readRequestBody(request) {
+async function readRequestBody(request: Request) {
   const { headers } = request
   const contentType = headers.get("content-type") || ""
 
   if (contentType.includes("application/json")) {
-    return await request.json();
+    return request.json();
```

Comment:
`return await` is unnecessary and functions exactly the same as just `return` in this situation.

The only reason you would ever `return await` is if your return statement is within a try/catch block, as you'd be able to catch a rejection from the promise you're awaiting.